src/operation/get-parameters-as-json-schema.js

```diff
@@ -415,7 +444,10 @@ function getRequestBody(operation, oas, globalDefaults) {
     examples.push({ examples: requestBody.examples });
   }
 
-  const cleanedSchema = constructSchema(requestBody.schema, examples, '', globalDefaults);
+  // We're cloning the request, and further below the component, schema because we've had issues with request schemas
+  // that were dereferenced being processed multiple times because their component is also processed.
+  const requestSchema = cloneObject(requestBody.schema);
```

Comment:
@Dashron @julshotal I discovered a bug while adding support for `items` to this work, and changing the example OAS I wrote to use `$ref` pointers, where schemas were sometimes being processed multiple times depending if they were used in a `$ref`. I've added these `cloneObject()` methods around in a few places where we're initially calling `constructSchema` to fix it.